src/tuntap/windows/c_tuntap_windows.cpp

```diff
@@ -48,6 +49,44 @@ void c_tuntap_windows_obj::async_receive_from_tun(unsigned char *const data, siz
 void c_tuntap_windows_obj::set_tun_parameters(const std::array<unsigned char, 16> &binary_address, int prefix_len, uint32_t mtu) {
 	_fact("Setting IPv6 address, prefixLen=" << prefix_len);
 	std::wstring human_name = get_human_name(m_guid);
+	NET_LUID luid = get_luid(human_name);
+	_fact("Setting address on human_name " << to_string(human_name));// << " luid=" << to_string(luid));
+																	 // remove old address
```

Comment:
comment that
1) it's copy paste from our previous commit (what commit?)
2) ---> link to MSDN or other docs